haiku-apps/minimizeall/minimizeall-1.0.0.recipe

```diff
@@ -1,15 +1,18 @@
 SUMMARY="Minimize all your apps with the press of a button!"
-DESCRIPTION="
+DESCRIPTION="
```

Comment:
You inserted (accidentally) a Control-Z at the end of this line, after **`DESCRIPTION="`**.

BTW, to check that your recipe does not have any unprintable char, you can run:
``
cat -v minimizeall-1.0.0.recipe | diff minimizeall-1.0.0.recipe -
```
If it outputs anything then you have unprintable chars.